src/NUnitFramework/framework/Constraints/CollectionItemsEqualConstraint.cs

```diff
@@ -34,10 +34,11 @@ namespace NUnit.Framework.Constraints
     /// </summary>
     public abstract class CollectionItemsEqualConstraint : CollectionConstraint
     {
+
         /// <summary>
         /// The NUnitEqualityComparer in use for this constraint
         /// </summary>
-        private readonly NUnitEqualityComparer _comparer = new NUnitEqualityComparer();
+        protected NUnitEqualityComparer Comparer { get; } = new NUnitEqualityComparer();
```

Comment:
Exposing this as new public API is no longer necessary if you delegate to `base` for all shadowing methods.